environment/elisp/draco-mode.el

```diff
@@ -466,11 +466,11 @@ function or method. Width is set by draco-code-comment-width.
 "Function for inserting a class desicription boilerplate.  Width is set
```

Comment:
I noticed that description is misspelled here, if you want to change it :)

Reply:
I'll fix it. Let me know when you are done reviewing.  I'll submit all the changes in a single push.